### DISCUSSION TITLE
Issue with Modal Vertical Centering Using top: 50% Causing Subpixel Rendering Blurriness

### DISCUSSION THREAD

**Discussion by KonstantinosVazaios** [rewindui/rewindui#123] [General]
I’ve encountered an issue with the Modal component when attempting to vertically center it using top: 50%. This method often results in subpixel rendering problems, causing the modal content (especially text) to appear blurry on certain displays or browsers due to fractional pixel calculations. Does anyone else, face the same issue?
I am providing 2 images as well of the Modal component being rendered at different viewport heights to see the blurry text
![image](https://github.com/user-attachments/assets/0e78d63a-a942-40f8-9810-66ab4c97d01a)
![image](https://github.com/user-attachments/assets/22eaab62-b638-4afd-aa31-c69b2ce9e431)

**Comment by NickDunas** [comment]
This is really odd! If you check on the docs this does not happen though: https://rewind-ui.dev/components/modal#position

Are you using the position prop or you are doing it with custom css?

**Reply by KonstantinosVazaios** [reply]
I agree it's extremely odd😅
It think same thing happens on documentation as well if you select position='center' and decrease the height of the viewport (till it provokes and overflow-y to the modal content), it will look like this (see image below).
If you select position top and check again the same viewport, it is not blurry.
It's weird because on the documentation this only happens when your modal has a scrollable overflow-y, while on my modal happens even if the content is not scrollable. 
This blurry behavior does not exist when selecting position top or bottom

![image](https://github.com/user-attachments/assets/3e3c12a4-eac9-4368-a7e9-2791ab418ce4)
